server/src/resolvers/Mutation/comment.js

```diff
@@ -45,7 +45,7 @@ const comment = {
       data.message = message;
     }
 
-    if (isPublic) {
+    if (isPublic !== undefined) {
```

Comment:
because `isPublic` can be `false` this prevented changing private comments to public